src/components/App/App.tsx

```diff
@@ -164,7 +170,7 @@ class App extends React.Component {
 
     const includeAds = !['/', '/api-keys'].includes(location.pathname);
     return (
-      <MuiThemeProvider muiTheme={getMuiTheme(darkBaseTheme, muiTheme)}>
+      <MuiThemeProvider muiTheme={getMuiTheme(darkBaseTheme, muiTheme as any)}>
```

Comment:
adjusting `muiTheme` props to conform typings is hard

Reply:
Yep, that's fine. We are likely moving away from material-ui anyway